nimare/meta/utils.py

```diff
@@ -275,14 +275,50 @@ def generate_input_fn():
     return niis
 
 
+def unique_rows(ar):
```

Comment:
This is straight from skimage, so I could import but  I don't want that as a dependency

Reply:
Is their license amenable to using their code directly? If so, then I'm all for just using their function without adding a dependency.

Reply:
```
Copyright (C) 2019, the scikit-image team
All rights reserved.

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are
met:

 1. Redistributions of source code must retain the above copyright
    notice, this list of conditions and the following disclaimer.
 2. Redistributions in binary form must reproduce the above copyright
    notice, this list of conditions and the following disclaimer in
    the documentation and/or other materials provided with the
    distribution.
 3. Neither the name of skimage nor the names of its contributors may be
    used to endorse or promote products derived from this software without
    specific prior written permission.
```

Looks like it just need to add the copyright

Reply:
Awesome! I'm happy to just use the code then.